homeassistant/components/todo/__init__.py

```diff
@@ -33,7 +33,15 @@
     ATTR_DUE,
     ATTR_DUE_DATE,
     ATTR_DUE_DATETIME,
+    ATTR_ITEM,
+    ATTR_RENAME,
+    ATTR_STATUS,
     DOMAIN,
+    SERVICE_ADD_ITEM,
+    SERVICE_GET_ITEMS,
+    SERVICE_REMOVE_COMPLETED_ITEMS,
+    SERVICE_REMOVE_ITEM,
+    SERVICE_UPDATE_ITEM,
```

Comment:
Instead introduce these services, should we use an enum?

I see that pattern in more modern integrations, and might not be a bad one.

Reply:
What's the benefit of an enum in this case?

Reply:
Having a single collection of services available in an enum

Reply:
LOL I actually just found one 🙈 

![CleanShot 2024-07-09 at 20 34 50@2x](https://github.com/home-assistant/core/assets/195327/3c1241c2-0edc-463a-bf16-87fbcd3bc4cc)

Maybe not as common after all...

WDYT?

Reply:
I mean, I don't have a strong opinion, but I do like consistency

Reply:
🤷‍♂️ Well apparently, this isn't consistent, the question is: Is this maybe something we want to make consistent?

Also, do we need these as constants?

Reply:
They are used in tests where we test the services, and I want to be able to use constants there